src/dispatchers/touchDispatcher.ts

```diff
@@ -0,0 +1,138 @@
+      this._event2Callback["touchend"] = this._processEndCallback;
+
+      this._broadcasters = [this._moveBroadcaster, this._startBroadcaster, this._endBroadcaster];
+    }
+
+    protected _getWrappedCallback(callback: Function): Core.BroadcasterCallback<Dispatcher.Touch> {
+      return () => callback(this.getLastTouchPosition());
+    }
+
+    /**
+     * Registers a callback to be called whenever the touch position changes,
+     * or removes the callback if `null` is passed as the callback.
+     *
+     * @param {any} key The key associated with the callback.
+     *                  Key uniqueness is determined by deep equality.
+     * @param {(p: Point) => any} callback A callback that takes the pixel position
+     *                                     in svg-coordinate-space. Pass `null`
+     *                                     to remove a callback.
+     * @return {Dispatcher.Mouse} The calling Dispatcher.Mouse.
+     */
+    public onTouchMove(key: any, callback: (p: Point) => any): Dispatcher.Touch {
+      this._setCallback(this._moveBroadcaster, key, callback);
+      return this;
+    }
+
+    /**
+     * Registers a callback to be called whenever a touch starts,
+     * or removes the callback if `null` is passed as the callback.
+     *
+     * @param {any} key The key associated with the callback.
+     *                  Key uniqueness is determined by deep equality.
+     * @param {(p: Point) => any} callback A callback that takes the pixel position
+     *                                     in svg-coordinate-space. Pass `null`
+     *                                     to remove a callback.
+     * @return {Dispatcher.Touch} The calling Dispatcher.Touch.
+     */
+    public onTouchStart(key: any, callback: TouchCallback): Dispatcher.Touch {
+      this._setCallback(this._startBroadcaster, key, callback);
+      return this;
+    }
+
+    /**
+     * Registers a callback to be called whenever a touch ends,
+     * or removes the callback if `null` is passed as the callback.
+     *
+     * @param {any} key The key associated with the callback.
+     *                  Key uniqueness is determined by deep equality.
+     * @param {(p: Point) => any} callback A callback that takes the pixel position
+     *                                     in svg-coordinate-space. Pass `null`
+     *                                     to remove a callback.
+     * @return {Dispatcher.Touch} The calling Dispatcher.Touch.
+     */
+    public onTouchEnd(key: any, callback: TouchCallback): Dispatcher.Touch {
+      this._setCallback(this._endBroadcaster, key, callback);
+      return this;
+    }
+
+    /**
+     * Computes the Touch position from the given event, and if successful
+     * calls broadcast() on the supplied Broadcaster.
+     */
+    private _measureAndBroadcast(e: TouchEvent, b: Core.Broadcaster<Dispatcher.Touch>) {
+      var touch = e.changedTouches[0];
```

Comment:
Added in [this commit](https://github.com/palantir/plottable/commit/4d396d0fb864be5a4eaed90e6287e070672956cd).